lib/doorkeeper/oauth/pre_authorization.rb

```diff
@@ -26,6 +27,7 @@ def initialize(server, attrs = {})
         @state                 = attrs[:state]
         @code_challenge        = attrs[:code_challenge]
         @code_challenge_method = attrs[:code_challenge_method]
+        @resource_owner        = resource_owner
```

Comment:
Let's add it also to attr_readers, maybe we wanna to access it in the future for some reason